devicon.json

```
@@ -5216,6 +5216,28 @@
         "color": "#83CD29",
         "aliases": []
     },
+    {
+        "name": "nodemon",
+        "altnames": [
+            "nodemonjs"
+        ],
+        "tags": [
+            "nodejs"
+        ],
```

Just a little suggestion.
```suggestion
        "tags": [
            "nodejs",
            "tool"
        ],
```

---

Would like to hear @Snailedlt's opinion on it as well.

---

Agreed :)
But can we also add the Javascript tag? Nodemon is built with Javascript, and for Javascript
```suggestion
        "tags": [
            "nodejs",
            "tool",
            "javascript"
        ],
```